.gitignore

```diff
@@ -4,7 +4,7 @@
 *.dll
 *.so
 *.dylib
-./ktf*
+ktf*
```

Comment:
are you sure? what if you create a source file with a "ktf" prefix?

Reply:
Yeah, I'm pretty sure we wont create files like that.